app/controllers/settings_controller.py

```diff
@@ -577,6 +591,23 @@ def _update_model_from_view(self) -> None:
             self.settings_dialog.todds_overwrite_checkbox.isChecked()
         )
 
+        # themes tab
+        if self.settings_dialog.enable_themes_checkbox.isChecked():
+            self.settings.enable_themes = True
+        else:
+            self.settings.enable_themes = False
+        # get theme names
+        selected_theme = self.settings_dialog.themes_combobox.currentText()
+        self.settings.theme = selected_theme
+        available_themes = [folder.name for folder in Themes.get_available_themes()]
```

Comment:
I wonder if it would be better to use an index based approach to store the themes. Or at least a tuple pair storing the index and the name. That way you avoid issues with duplicate names, string formatting, etc.

Reply:
Not Everyone is Going To use the same set of themes also most people install less then 10 themes max not 100 themes so i think its ok for now